spell/collateral-onboarding-checklist.md

```diff
@@ -107,8 +107,8 @@
         * [ ] `LinearDecrease`
           * [ ] `calc.tau` in seconds (e.g. 250 days = 21_600_000)
         * [ ] `StairstepExponentialDecrease`
-          * [ ] `calc.cut` in seconds (e.g. 120 seconds)
-          * [ ] `calc.step` in basis points (e.g. 99% = 99_00)
+          * [ ] `calc.step` in seconds (e.g. 120 seconds)
+          * [ ] `calc.cut` in basis points (e.g. 99% = 99_00)
```

Comment:
the order follows the fileable order in dss [abaci](https://github.com/makerdao/dss/blob/master/src/abaci.sol#L122)

Reply:
but I see the issue, they were reverted

Reply:
I would suggest to keep `cut`, `step` order though

Reply:
actually is good, as it reflects the parameters order of the dss-exec-lib func, `setStairstepExponentialDecrease`.
I guess I got confused by that, as then it follows dss order in order of fileable operations.